src/plugins/data_table/filter_view.js

```diff
@@ -119,16 +123,13 @@ var DataTableFilter = (function(options) {
     },
 
     disableFilter: function(errorMessage) {
+      this.errorMessage = errorMessage;
       if (!this.enabled) return;
```

Comment:
this line can just be removed now.. it's harmless to simply do L128 idempotently and not have an early return here